packages/zowe-explorer/src/uss/actions.ts

```diff
@@ -64,7 +64,6 @@ export async function createUSSNode(node: IZoweUSSTreeNode, ussFileProvider: IZo
             filePath = `${filePath}/${name}`;
             await ZoweExplorerApiRegister.getUssApi(node.getProfile()).create(filePath, nodeType);
             if (isTopLevel) {
-                await Profiles.getInstance().refresh(ZoweExplorerApiRegister.getInstance());
```

Comment:
Not really a suggestion, just curious to know why we've removed this 😅

Reply:
The next line calls the `refreshAll` method, and the first thing that `refreshAll` does is call `await Profiles.getInstance().refresh(ZoweExplorerApiRegister.getInstance());` so we were doing it twice 😁